test/java/src/com/ibm/streamsx/topology/test/api/CompGenerationTest.java

```diff
@@ -75,6 +77,7 @@ public void nestedUDPWithBroadcast() throws Exception {
     @Test
     public void multipleInputPorts() throws Exception {
         assumeTrue(SC_OK);
+        assumeTrue(!isMainRun());
```

Comment:
Is this required? SC_OK (the check above) will rule out embedded?

Reply:
It didn't work without it, yes.

Reply:
When running `unittest.quick` it would still try to execute the test and fail on its condition.